roles/edpm_libvirt/tasks/post-install.yml

```diff
@@ -43,3 +43,10 @@
   ansible.builtin.include_role:
       name: osp.edpm.edpm_nftables
       tasks_from: "configure.yml"
+- name: Reload firewall for new vnc rule
+  tags:
+    - install
+    - post-libvirt
+  ansible.builtin.include_role:
+      name: osp.edpm.edpm_nftables
+      tasks_from: "run.yml"
```

Comment:
ok yes this is a bug intoduced in the porting.

hum ok I'm not quite sure how this was working before but yes this is required.

Reply:
by the way it would be nice to add a molecule test to assert that the rule is actually reloaded. i.e. to exentd https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_libvirt/molecule/default/test-helpers/verify_firewall.yaml
to also grep the output of the nft rules.

Reply:
Here's that molecule test https://github.com/openstack-k8s-operators/edpm-ansible/pull/306